quota/etcd/quota_provider.go

```diff
@@ -58,8 +58,7 @@ func newEtcdQuotaManager() (quota.Manager, error) {
 		return nil, fmt.Errorf("failed to connect to etcd at %v: %v", *Servers, err)
 	}
 
-	var qm quota.Manager
-	qm = etcdqm.New(client)
+	var qm quota.Manager = etcdqm.New(client)
```

Comment:
I assume we're deliberately dropping the specific type here. Is that necessary?

Reply:
It needs to be this way to support the `qm = cachedQM` line below.